packages/plugins/src/api/index.js

```diff
@@ -105,6 +108,13 @@ export function registerPlugin( name, settings ) {
 		);
 		return null;
 	}
+	const { priority = 10 } = settings;
+	if ( priority !== 0 && ( priority.length === 0 || typeof priority !== 'number' ) ) {
```

Comment:
Can we use [Number.isInteger()](https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Number/isInteger) instead?

Reply:
Done.